hack/tools/test_module.sh

```diff
@@ -0,0 +1,232 @@
+
+# if [ $2 == "dev" ]
+# then
+#     cd ${PATH_TO_LOCAL_FYBRIK}
+#     helm dependency update charts/vault
+#     helm install vault charts/vault --create-namespace -n fybrik-system \
+#         --set "vault.injector.enabled=false" \
+#         --set "vault.server.dev.enabled=true" \
+#         --values charts/vault/env/dev/vault-single-cluster-values.yaml
+#     kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
+
+# else
+    helm install vault fybrik-charts/vault --create-namespace -n fybrik-system \
+        --set "vault.injector.enabled=false" \
+        --set "vault.server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/fybrik/fybrik/v0.5.3/charts/vault/env/dev/vault-single-cluster-values.yaml
+    kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=220s
+# fi
+
+
+
+helm install fybrik-crd fybrik-charts/fybrik-crd -n fybrik-system --version v$fybrikVersion --wait
+helm install fybrik fybrik-charts/fybrik -n fybrik-system --version v$fybrikVersion --wait
+
+
+# cd ${PATH_TO_LOCAL_FYBRIK}
+# helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
+# helm install fybrik charts/fybrik --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait
+
+
+# hello-world-module
+
+#datashim
+kubectl apply -f https://raw.githubusercontent.com/datashim-io/datashim/master/release-tools/manifests/dlf.yaml
+kubectl wait --for=condition=ready pods -l app.kubernetes.io/name=dlf -n dlf --timeout=500s
+
+
+sleep 10
+# kubectl apply -f hello-world-module.yaml -n fybrik-system
+kubectl apply -f https://github.com/fybrik/hello-world-module/releases/download/v$moduleVersion/hello-world-module.yaml -n fybrik-system
+
+
+# Notebook sample
+
+kubectl create namespace fybrik-notebook-sample
+kubectl config set-context --current --namespace=fybrik-notebook-sample
+
+#localstack
+helm repo add localstack-charts https://localstack.github.io/helm-charts
+helm install localstack localstack-charts/localstack --set startServices="s3" --set service.type=ClusterIP
+kubectl wait --for=condition=ready --all pod -n fybrik-notebook-sample --timeout=400s
+
+kubectl port-forward svc/localstack 4566:4566 &
+
+export ENDPOINT="http://127.0.0.1:4566"
+export BUCKET="demo"
+export OBJECT_KEY="PS_20174392719_1491204439457_log.csv"
+export FILEPATH="$WORKING_DIR/PS_20174392719_1491204439457_log.csv"
+aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY} && aws --endpoint-url=${ENDPOINT} s3api create-bucket --bucket ${BUCKET} && aws --endpoint-url=${ENDPOINT} s3api put-object --bucket ${BUCKET} --key ${OBJECT_KEY} --body ${FILEPATH}
+
+
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: paysim-csv
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+EOF
+
+
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
```

Comment:
I think `Asset` in 0.5.0 version is different from 0.6.0 version so we will need two versions.